voctogui/lib/videodisplay.py

```diff
@@ -103,24 +111,29 @@ def __init__(self, drawing_area, port, width=None, height=None,
         # If an Audio-Path is required,
         # add an Audio-Path through a level-Element
         if self.level_callback or play_audio:
-            pipeline += """
-                demux. !
-                {acaps} !
-                queue !
-                level name=lvl interval=50000000 !
-            """
-
-            # If Playback is requested, push fo pulseaudio
-            if play_audio:
+            for audiostream in range(0, audiostreams):
                 pipeline += """
-                    pulsesink
-                """
-
-            # Otherwise just trash the Audio
-            else:
+                    demux.audio_{audiostream} !
+                """.format(audiostream=audiostream)
+                pipeline += """
+                    {acaps} !
+                    queue !
+                    """
                 pipeline += """
-                    fakesink
-                """
+                    level name=lvl_{audiostream} interval=50000000 !
+                """.format(audiostream=audiostream)
+
+                # If Playback is requested, push fo pulseaudio
+                if play_audio:
```

Comment:
This would create multiple pulsesinks when multiple vu-meters are activated, playing back all languages at once. We usually only want to playback the first stream and dump the remaining ones into a fakesink